src/azurelinux/3.0/net9.0/webassembly/amd64/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN tdnf update -y \
         # WebAssembly build dependencies
         which \
         nodejs \
+        npm \
```

Comment:
probably should add it to the net8 image too?

Reply:
Ah I was searching with `nodejs \`, didn't realized in net8.0 files it was the last one in the list 😅